secureli/actions/action.py

```diff
@@ -98,8 +101,35 @@ def verify_install(
 
         config = SecureliConfig() if reset else self.action_deps.secureli_config.load()
 
-        if not config.languages or not config.version_installed:
-            return self._install_secureli(folder_path, always_yes)
+        try:
+            languages = self._detect_languages(folder_path)
+        except (ValueError, LanguageNotSupportedError) as e:
+            if config.languages and config.version_installed:
+                self.action_deps.echo.warning(
+                    f"Newly detected languages are unsupported by seCureLI"
```

Comment:
Yes, since scan and init run the same language detection function there will be a few cases like this. I have it printing a warning instead of being a failure like on a new install, but I think there needs to be future enhancements to better split the scan and init language detection logic.

Reply:
There are a lot of similar logs that happen each scan so I'm going to create an issue to better abstract scan and init.